### ISSUE TITLE
week4: Reusage of ReplayBuffer

### ISSUE THREAD

**Issue by NickVeld** [yandexdataschool/Practical_RL#147]
The task offer to copy and paste the implementation of the ReplayBuffer from week3
The task from week3 offer to choose any data structure for the implementation.
If np.ndarray was picked and tests with ReplayBuffer raises exception because tests call _storage directly, observe it as a list and get something other from that must be.

**Comment by justheuristic** [comment]
Yeah, that wold be great. We'll implement the solution once the deadline expires (in order to avoid merge conflicts for students).
